src/components/LogoCard.module.css

```diff
@@ -1,4 +1,14 @@
+.outer {
```

Comment:
This new outer class gives a static outer size (across hover and click CSS changes) to prevent shifting the other buttons on hover or click.